spec/workers/link_check_worker_spec.rb

```
@@ -95,14 +97,16 @@
           end
         end
 
-        context 'exceptions' do
+        context 'when there is an exception' do
           it 'handles throttling error' do
```

Is this testing exceptions are swallowed? If so could it be `it 'swallows ThrottleLimitError'`

---

Fair point

---

Done!